_data/versions.yml

```diff
@@ -1,3 +1,103 @@
+v3.1:
+- title: v3.1.0-rc1
+  note: |
+  18 March 2018
```

Comment:
Maybe bump to 21 March? I am not sure but seems like that will be the day.

Reply:
It should be Mar 20: https://github.com/projectcalico/calico/releases/tag/v3.1.0-rc1